Doras/src/app/roles/roles.component.html

```diff
@@ -1,51 +1,61 @@
 <div class='container-fluid' data-interval="false">
-    <div class='col-12'>
-        <ngb-carousel data-interval="false" class="carouselTop" (slide)="onSlide($event)">
-            <li *ngFor ="let band of endArray; let i = index">
-                <ng-template ngbSlide>
+  <div class='col-12'>
+    <ngb-carousel data-interval="false" class="carouselTop" (slide)="onSlide($event)">
+      <li *ngFor="let band of jobBandArray; let i = index">
+        <ng-template ngbSlide>
+          <table class="tableBand">
+            <tr>
+              <th class="th1" scope="col">{{jobBandArray[i][0].bandName}}
+                <li *ngFor="let job1 of jobs1">
+                  <h4 *ngIf="job1; else notShowBlock">{{job1.title}}</h4>
+                </li>
+              </th>
+              <th align="center" scope="col">{{jobBandArray[i][1].bandName}}
+                <li *ngFor="let job2 of jobs2">
+                  <h4 *ngIf="job2; else notShowBlock">{{job2.title}}</h4>
+                </li>
+              </th>
+              <th align="center" scope="col">{{jobBandArray[i][2].bandName}}
+                <li *ngFor="let job3 of jobs3">
+                  <h4 *ngIf="job3; else notShowBlock">{{job3.title}}</h4>
+                </li>
+              </th>
+              <ng-template #notShowBlock>
+                <h4>No Role</h4>
+              </ng-template>
+            </tr>
+            <tr>
+              <td>
 
-                    <table class="tableBand">
-                        <tr>
-                            <th
-                                class="th1" scope="col">{{endArray[i][0].bandName}}
-                                <h4 *ngIf="job1Title; else notShowBlock">{{job1Title}}</h4>
-                            </th>
-                                <th align="center" scope="col">{{endArray[i][1].bandName}}
-                                <h4 *ngIf="job2Title; else notShowBlock">{{job2Title}}</h4>
-                            </th>
-                                <th align="center" scope="col">{{endArray[i][2].bandName}}
-                                <h4 *ngIf="job3Title; else notShowBlock">{{job3Title}}</h4>
-                                </th>
-                                <ng-template #notShowBlock>
-                                  <h4>No Role</h4>
-                                </ng-template>
-                        </tr>
-                        <tr>
-                            <td>
+                <p></p>
+                <button class="buttonTable" (click)="openResponsibility1()">Responsibilities</button>
```

Comment:
`openResponsibility1` requires a better name, same as jobs
`openFirstResponsibility` would be better and jobs respectively